.github/workflows/itest.yml

```diff
@@ -10,11 +10,13 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # os: [ macos-latest, windows-latest, ubuntu-latest ]
-        os: [ windows-latest, ubuntu-latest ]
+        os: [ macos-latest, windows-latest, ubuntu-latest ]
         java: [ '11' ]
     steps:
       - uses: actions/checkout@v2
+      - name: Setup SBT
+        if: matrix.os == 'macos-latest'
+        run: brew install sbt
```

Comment:
The latest github macos runner no longer includes sbt according to [this issue](https://github.com/actions/setup-java/issues/627). We have to install sbt manually on macos.